src/components/ClusterNodes/NodeDetails.tsx

```diff
@@ -191,71 +190,34 @@ export default function NodeDetails({ imageList, isSuperAdmin, namespaceList }:
-                        <Edit className="icon-dim-16 mt-2 mr-5 edit-yaml-icon" />
-                        {NODE_DETAILS_TABS.yaml}
-                    </div>
-                    {selectedTabIndex == 1 && <div className="node-details__active-tab" />}
-                </li>
-                <li className={`tab-list__tab ${cursorValue}`} data-tab-index="2" onClick={changeNodeTab}>
-                    <div className={`mb-6 fs-13 tab-hover${selectedTabIndex == 2 ? ' fw-6 active' : ' fw-4'}`}>
-                        {NODE_DETAILS_TABS.nodeConditions}
-                    </div>
-                    {selectedTabIndex == 2 && <div className="node-details__active-tab" />}
-                </li>
-                {isSuperAdmin && (
-                    <li className="tab-list__tab cursor" data-tab-index="3" onClick={changeNodeTab}>
-                        <div
-                            className={`mb-6 flexbox fs-13 tab-hover${
-                                selectedTabIndex == 3 ? ' fw-6 active' : ' fw-4'
-                            }`}
-                            data-testid="node-details-terminal-tab"
-                        >
-                            <TerminalIcon className="icon-dim-16 mt-2 mr-5 terminal-icon" />
-                            {NODE_DETAILS_TABS.debug}
-                        </div>
-                        {selectedTabIndex == 3 && <div className="node-details__active-tab" />}
-                    </li>
-                )}
-            </ul>
-        )
-    }
-
-    const { breadcrumbs } = useBreadcrumb(
-        {
-            alias: {
-                clusters: {
-                    component: 'Clusters',
-                    linked: true,
-                },
-                ':clusterId': {
-                    component: nodeDetail?.clusterName,
-                    linked: true,
-                },
-                ':nodeName': {
-                    component: isNodeAuto ? 'Selecting node' : nodeName,
-                    linked: false,
-                },
-            },
-        },
-        [clusterId, nodeName, nodeDetail],
-    )
-
-    const renderBreadcrumbs = (): JSX.Element => {
+        const cursorValue = 'cursor'
         return (
-            <span className={isNodeAuto ? 'dc__loading-dots' : ''}>
-                <BreadCrumb breadcrumbs={breadcrumbs} />
-            </span>
+            <div className='pl-20 flex dc__border-bottom'>
+                <div className='flex left w-100'>
+                    <ul role="tablist" className="tab-list pt-6">
+                        <li className={`tab-list__tab ${cursorValue}`} data-tab-index="0" onClick={changeNodeTab}>
+                            <div className={`mb-6 fs-12 tab-hover${selectedTabIndex == 0 ? ' fw-6 active' : ' fw-4'}`}>
+                                {NODE_DETAILS_TABS.summary}
+                            </div>
+                            {selectedTabIndex == 0 && <div className="node-details__active-tab" />}
```

Comment:
selectedTabIndex == 0 Repace == to === everywhere, 0 must be replace to constant in case holding some value